core/models.py

```diff
@@ -1,3 +1,29 @@
 from django.db import models
+from django.contrib.auth.models import User
 
-# Create your models here.
+
+class Client(models.Model):
+    user = models.ForeignKey(User, on_delete=models.PROTECT)
+    name = models.CharField(max_length=100)
+    surname = models.CharField(max_length=100)
+    email = models.EmailField(max_length=100)
+    telephone = models.CharField(max_length=15)
+    cpf = models.CharField(max_length=14)
+
+
+class Loan(models.Model):
+    user = models.ForeignKey(User, on_delete=models.PROTECT)
+    client = models.ForeignKey(Client, on_delete=models.PROTECT)
+    amount = models.DecimalField(decimal_places=2, max_digits=10)
+    term = models.IntegerField()
+    rate = models.FloatField()
+    date = models.DateField(auto_now_add=True)
+    installment = models.DecimalField(decimal_places=2, max_digits=10, blank=True, null=True)
+
+
+class Payment(models.Model):
+    loan = models.ForeignKey(Loan, on_delete=models.PROTECT)
+    user = models.ForeignKey(User, on_delete=models.PROTECT)
```

Comment:
Do we need this field? Only the assigned user will be able to manage this data. Besides, this info will be available in the Loan data.